Cannot build SDL2 using CMake when targeting UWP

```
SDL.obj : error LNK2019: unresolved external symbol _exit referenced in function SDL_ExitProcess [C:\Users\Sahil\SDL\bu
ild-winrt-2nd\SDL2.vcxproj]
SDL_winrtvideo.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\build-winrt-2nd\S
DL2.vcxproj]
SDL_sysfilesystem.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\build-winrt-2n
d\SDL2.vcxproj]
msvcrtd.lib(new_scalar_nothrow.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\
build-winrt-2nd\SDL2.vcxproj]
SDL_winrtmessagebox.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\build-winrt-
2nd\SDL2.vcxproj]
SDL_winrtmouse.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\build-winrt-2nd\S
DL2.vcxproj]
SDL_winrtopengles.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\build-winrt-2n
d\SDL2.vcxproj]
SDL_winrtpointerinput.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\build-winr
t-2nd\SDL2.vcxproj]
SDL_wasapi_winrt.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\build-winrt-2nd
\SDL2.vcxproj]
SDL_render_winrt.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\build-winrt-2nd
\SDL2.vcxproj]
SDL_winrtgamebar.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\build-winrt-2nd
\SDL2.vcxproj]
SDL_winrtkeyboard.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\build-winrt-2n
d\SDL2.vcxproj]
SDL_winrtapp_common.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\build-winrt-
2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\build-winr
t-2nd\SDL2.vcxproj]
SDL_winrtapp_xaml.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\build-winrt-2n
d\SDL2.vcxproj]
SDL_sysurl.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4 [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.
vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2001: unresolved external symbol _purecall [C:\Users\Sahil\SDL\build-winrt-2nd\SDL
2.vcxproj]
SDL_wasapi_winrt.obj : error LNK2001: unresolved external symbol _purecall [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcx
proj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol __imp__invalid_parameter referenced in function "
void * __cdecl std::_Allocate_manually_vector_aligned<struct std::_Default_allocate_traits>(unsigned __int64)" (??$_All
ocate_manually_vector_aligned@U_Default_allocate_traits@std@@@std@@YAPEAX_K@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2
.vcxproj]
SDL_wasapi_winrt.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter [C:\Users\Sahil\SDL\build-win
rt-2nd\SDL2.vcxproj]
SDL_sysfilesystem.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter [C:\Users\Sahil\SDL\build-wi
nrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol __imp__CrtDbgReport referenced in function "void
* __cdecl std::_Allocate_manually_vector_aligned<struct std::_Default_allocate_traits>(unsigned __int64)" (??$_Allocate
_manually_vector_aligned@U_Default_allocate_traits@std@@@std@@YAPEAX_K@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxp
roj]
SDL_wasapi_winrt.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport [C:\Users\Sahil\SDL\build-winrt-2n
d\SDL2.vcxproj]
SDL_sysfilesystem.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport [C:\Users\Sahil\SDL\build-winrt-2
nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol __imp__CrtDbgReportW referenced in function "prot
ected: __cdecl Concurrency::details::_RefCounter::_RefCounter(long)" (??0_RefCounter@details@Concurrency@@IEAA@J@Z) [C:
\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_wasapi_winrt.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReportW [C:\Users\Sahil\SDL\build-winrt-2
nd\SDL2.vcxproj]
vccorlibd.lib(init.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW [C:\Users\Sahil\SDL\build-winr
t-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl std::_Lock
it::_Lockit(int)" (__imp_??0_Lockit@std@@QEAA@H@Z) referenced in function "public: void __cdecl std::_Container_base12:
:_Orphan_all(void)" (?_Orphan_all@_Container_base12@std@@QEAAXXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_wasapi_winrt.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) public: __cdecl std::_Lockit::_
Lockit(int)" (__imp_??0_Lockit@std@@QEAA@H@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_sysfilesystem.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) public: __cdecl std::_Lockit::
_Lockit(int)" (__imp_??0_Lockit@std@@QEAA@H@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl std::_Lock
it::~_Lockit(void)" (__imp_??1_Lockit@std@@QEAA@XZ) referenced in function "public: void __cdecl std::_Container_base12
::_Orphan_all(void)" (?_Orphan_all@_Container_base12@std@@QEAAXXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_wasapi_winrt.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) public: __cdecl std::_Lockit::~
_Lockit(void)" (__imp_??1_Lockit@std@@QEAA@XZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_sysfilesystem.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) public: __cdecl std::_Lockit::
~_Lockit(void)" (__imp_??1_Lockit@std@@QEAA@XZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol __imp_abort referenced in function "void __cdecl
std::call_once<class <lambda_0e64988457575224279dcb6d61e61bdd> >(struct std::once_flag &,class <lambda_0e64988457575224
279dcb6d61e61bdd> &&)" (??$call_once@V<lambda_0e64988457575224279dcb6d61e61bdd>@@$$V@std@@YAXAEAUonce_flag@0@$$QEAV<lam
bda_0e64988457575224279dcb6d61e61bdd>@@@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol __std_exception_copy referenced in function "publ
ic: __cdecl std::exception::exception(class std::exception const &)" (??0exception@std@@QEAA@AEBV01@@Z) [C:\Users\Sahil
\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_wasapi_winrt.obj : error LNK2001: unresolved external symbol __std_exception_copy [C:\Users\Sahil\SDL\build-winrt-2
nd\SDL2.vcxproj]
SDL_sysfilesystem.obj : error LNK2001: unresolved external symbol __std_exception_copy [C:\Users\Sahil\SDL\build-winrt-
2nd\SDL2.vcxproj]
msvcrtd.lib(throw_bad_alloc.obj) : error LNK2001: unresolved external symbol __std_exception_copy [C:\Users\Sahil\SDL\b
uild-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol __std_exception_destroy referenced in function "p
ublic: virtual __cdecl std::exception::~exception(void)" (??1exception@std@@UEAA@XZ) [C:\Users\Sahil\SDL\build-winrt-2n
d\SDL2.vcxproj]
SDL_wasapi_winrt.obj : error LNK2001: unresolved external symbol __std_exception_destroy [C:\Users\Sahil\SDL\build-winr
t-2nd\SDL2.vcxproj]
SDL_sysfilesystem.obj : error LNK2001: unresolved external symbol __std_exception_destroy [C:\Users\Sahil\SDL\build-win
rt-2nd\SDL2.vcxproj]
msvcrtd.lib(throw_bad_alloc.obj) : error LNK2001: unresolved external symbol __std_exception_destroy [C:\Users\Sahil\SD
L\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "void __cdecl __ExceptionPtrCreate(void *)" (?__E
xceptionPtrCreate@@YAXPEAX@Z) referenced in function "public: __cdecl std::exception_ptr::exception_ptr(void)" (??0exce
ption_ptr@std@@QEAA@XZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "void __cdecl __ExceptionPtrDestroy(void *)" (?__
ExceptionPtrDestroy@@YAXPEAX@Z) referenced in function "public: __cdecl std::exception_ptr::~exception_ptr(void)" (??1e
xception_ptr@std@@QEAA@XZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "void __cdecl __ExceptionPtrCopy(void *,void cons
t *)" (?__ExceptionPtrCopy@@YAXPEAXPEBX@Z) referenced in function "public: __cdecl std::exception_ptr::exception_ptr(cl
ass std::exception_ptr const &)" (??0exception_ptr@std@@QEAA@AEBV01@@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxpro
j]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "void __cdecl __ExceptionPtrCurrentException(void
 *)" (?__ExceptionPtrCurrentException@@YAXPEAX@Z) referenced in function "public: static class std::exception_ptr __cde
cl std::exception_ptr::_Current_exception(void)" (?_Current_exception@exception_ptr@std@@SA?AV12@XZ) [C:\Users\Sahil\SD
L\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "void __cdecl std::_Xbad_alloc(void)" (?_Xbad_all
oc@std@@YAXXZ) referenced in function "protected: void __cdecl std::basic_stringbuf<char,struct std::char_traits<char>,
class std::allocator<char> >::_Init(char const *,unsigned __int64,int)" (?_Init@?$basic_stringbuf@DU?$char_traits@D@std
@@V?$allocator@D@2@@std@@IEAAXPEBD_KH@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "void __cdecl std::_Xlength_error(char const *)"
(?_Xlength_error@std@@YAXPEBD@Z) referenced in function "void __cdecl std::_Xlen_string(void)" (?_Xlen_string@std@@YAXX
Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_wasapi_winrt.obj : error LNK2001: unresolved external symbol "void __cdecl std::_Xlength_error(char const *)" (?_Xl
ength_error@std@@YAXPEBD@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_sysfilesystem.obj : error LNK2001: unresolved external symbol "void __cdecl std::_Xlength_error(char const *)" (?_X
length_error@std@@YAXPEBD@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "void __cdecl std::_Xout_of_range(char const *)"
(?_Xout_of_range@std@@YAXPEBD@Z) referenced in function "public: static void __cdecl std::_String_val<struct std::_Simp
le_types<char> >::_Xran(void)" (?_Xran@?$_String_val@U?$_Simple_types@D@std@@@std@@SAXXZ) [C:\Users\Sahil\SDL\build-win
rt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "void __cdecl std::_Xbad_function_call(void)" (?_
Xbad_function_call@std@@YAXXZ) referenced in function "public: unsigned char __cdecl std::_Func_class<unsigned char>::o
perator()(void)const " (??R?$_Func_class@E$$V@std@@QEBAEXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol _Mtx_init_in_situ referenced in function "public:
 __cdecl std::_Mutex_base::_Mutex_base(int)" (??0_Mutex_base@std@@QEAA@H@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vc
xproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol _Mtx_destroy_in_situ referenced in function "publ
ic: __cdecl std::_Mutex_base::~_Mutex_base(void)" (??1_Mutex_base@std@@QEAA@XZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL
2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol _Mtx_lock referenced in function "public: void __
cdecl std::_Mutex_base::lock(void)" (?lock@_Mutex_base@std@@QEAAXXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol _Mtx_unlock referenced in function "public: void
__cdecl std::_Mutex_base::unlock(void)" (?unlock@_Mutex_base@std@@QEAAXXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcx
proj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol _Cnd_init_in_situ referenced in function "public:
 __cdecl std::condition_variable::condition_variable(void)" (??0condition_variable@std@@QEAA@XZ) [C:\Users\Sahil\SDL\bu
ild-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol _Cnd_destroy_in_situ referenced in function "publ
ic: __cdecl std::condition_variable::~condition_variable(void)" (??1condition_variable@std@@QEAA@XZ) [C:\Users\Sahil\SD
L\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol _Cnd_wait referenced in function "public: void __
cdecl std::condition_variable::wait(class std::unique_lock<class std::mutex> &)" (?wait@condition_variable@std@@QEAAXAE
AV?$unique_lock@Vmutex@std@@@2@@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol _Cnd_broadcast referenced in function "public: vo
id __cdecl std::condition_variable::notify_all(void)" (?notify_all@condition_variable@std@@QEAAXXZ) [C:\Users\Sahil\SDL
\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "void __cdecl std::_Throw_C_error(int)" (?_Throw_
C_error@std@@YAXH@Z) referenced in function "int __cdecl std::_Check_C_return(int)" (?_Check_C_return@std@@YAHH@Z) [C:\
Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: bool __cdecl std::
ios_base::eof(void)const " (__imp_?eof@ios_base@std@@QEBA_NXZ) referenced in function "void __cdecl WINRT_SetDisplayOri
entationsPreference(void *,char const *,char const *,char const *)" (?WINRT_SetDisplayOrientationsPreference@@YAXPEAXPE
BD11@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) protected: __cdecl std::ba
sic_streambuf<char,struct std::char_traits<char> >::basic_streambuf<char,struct std::char_traits<char> >(void)" (__imp_
??0?$basic_streambuf@DU?$char_traits@D@std@@@std@@IEAA@XZ) referenced in function "public: __cdecl std::basic_stringbuf
<char,struct std::char_traits<char>,class std::allocator<char> >::basic_stringbuf<char,struct std::char_traits<char>,cl
ass std::allocator<char> >(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > cons
t &,int)" (??0?$basic_stringbuf@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@QEAA@AEBV?$basic_string@DU?$char_traits@D
@std@@V?$allocator@D@2@@1@H@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl st
d::basic_streambuf<char,struct std::char_traits<char> >::~basic_streambuf<char,struct std::char_traits<char> >(void)" (
__imp_??1?$basic_streambuf@DU?$char_traits@D@std@@@std@@UEAA@XZ) referenced in function "int `public: __cdecl std::basi
c_stringbuf<char,struct std::char_traits<char>,class std::allocator<char> >::basic_stringbuf<char,struct std::char_trai
ts<char>,class std::allocator<char> >(class basic_stringbuf<char,struct std::char_traits<char>,class std::allocator<cha
r> >::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,int)'::`1'::dtor$0" (?dtor$0
@?0???0?$basic_stringbuf@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V
?$allocator@D@2@@1@H@Z@4HA) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl std::b
asic_streambuf<char,struct std::char_traits<char> >::sbumpc(void)" (__imp_?sbumpc@?$basic_streambuf@DU?$char_traits@D@s
td@@@std@@QEAAHXZ) referenced in function "class std::basic_istream<char,struct std::char_traits<char> > & __cdecl std:
:getline<char,struct std::char_traits<char>,class std::allocator<char> >(class std::basic_istream<char,struct std::char
_traits<char> > &&,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > &,char)" (??
$getline@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@YAAEAV?$basic_istream@DU?$char_traits@D@std@@@0@$$QEAV10@AEAV?$b
asic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@0@D@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl std::b
asic_streambuf<char,struct std::char_traits<char> >::sgetc(void)" (__imp_?sgetc@?$basic_streambuf@DU?$char_traits@D@std
@@@std@@QEAAHXZ) referenced in function "class std::basic_istream<char,struct std::char_traits<char> > & __cdecl std::g
etline<char,struct std::char_traits<char>,class std::allocator<char> >(class std::basic_istream<char,struct std::char_t
raits<char> > &&,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > &,char)" (??$g
etline@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@YAAEAV?$basic_istream@DU?$char_traits@D@std@@@0@$$QEAV10@AEAV?$bas
ic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@0@D@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl std::b
asic_streambuf<char,struct std::char_traits<char> >::snextc(void)" (__imp_?snextc@?$basic_streambuf@DU?$char_traits@D@s
td@@@std@@QEAAHXZ) referenced in function "class std::basic_istream<char,struct std::char_traits<char> > & __cdecl std:
:getline<char,struct std::char_traits<char>,class std::allocator<char> >(class std::basic_istream<char,struct std::char
_traits<char> > &&,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > &,char)" (??
$getline@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@YAAEAV?$basic_istream@DU?$char_traits@D@std@@@0@$$QEAV10@AEAV?$b
asic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@0@D@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) protected: char * __cdecl
std::basic_streambuf<char,struct std::char_traits<char> >::eback(void)const " (__imp_?eback@?$basic_streambuf@DU?$char_
traits@D@std@@@std@@IEBAPEADXZ) referenced in function "protected: void __cdecl std::basic_stringbuf<char,struct std::c
har_traits<char>,class std::allocator<char> >::_Tidy(void)" (?_Tidy@?$basic_stringbuf@DU?$char_traits@D@std@@V?$allocat
or@D@2@@std@@IEAAXXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) protected: char * __cdecl
std::basic_streambuf<char,struct std::char_traits<char> >::gptr(void)const " (__imp_?gptr@?$basic_streambuf@DU?$char_tr
aits@D@std@@@std@@IEBAPEADXZ) referenced in function "protected: virtual int __cdecl std::basic_stringbuf<char,struct s
td::char_traits<char>,class std::allocator<char> >::overflow(int)" (?overflow@?$basic_stringbuf@DU?$char_traits@D@std@@
V?$allocator@D@2@@std@@MEAAHH@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) protected: char * __cdecl
std::basic_streambuf<char,struct std::char_traits<char> >::pptr(void)const " (__imp_?pptr@?$basic_streambuf@DU?$char_tr
aits@D@std@@@std@@IEBAPEADXZ) referenced in function "protected: void __cdecl std::basic_stringbuf<char,struct std::cha
r_traits<char>,class std::allocator<char> >::_Tidy(void)" (?_Tidy@?$basic_stringbuf@DU?$char_traits@D@std@@V?$allocator
@D@2@@std@@IEAAXXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) protected: char * __cdecl
std::basic_streambuf<char,struct std::char_traits<char> >::egptr(void)const " (__imp_?egptr@?$basic_streambuf@DU?$char_
traits@D@std@@@std@@IEBAPEADXZ) referenced in function "protected: void __cdecl std::basic_stringbuf<char,struct std::c
har_traits<char>,class std::allocator<char> >::_Tidy(void)" (?_Tidy@?$basic_stringbuf@DU?$char_traits@D@std@@V?$allocat
or@D@2@@std@@IEAAXXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) protected: void __cdecl st
d::basic_streambuf<char,struct std::char_traits<char> >::gbump(int)" (__imp_?gbump@?$basic_streambuf@DU?$char_traits@D@
std@@@std@@IEAAXH@Z) referenced in function "protected: virtual int __cdecl std::basic_stringbuf<char,struct std::char_
traits<char>,class std::allocator<char> >::pbackfail(int)" (?pbackfail@?$basic_stringbuf@DU?$char_traits@D@std@@V?$allo
cator@D@2@@std@@MEAAHH@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) protected: void __cdecl st
d::basic_streambuf<char,struct std::char_traits<char> >::setg(char *,char *,char *)" (__imp_?setg@?$basic_streambuf@DU?
$char_traits@D@std@@@std@@IEAAXPEAD00@Z) referenced in function "protected: void __cdecl std::basic_stringbuf<char,stru
ct std::char_traits<char>,class std::allocator<char> >::_Init(char const *,unsigned __int64,int)" (?_Init@?$basic_strin
gbuf@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@IEAAXPEBD_KH@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) protected: char * __cdecl
std::basic_streambuf<char,struct std::char_traits<char> >::epptr(void)const " (__imp_?epptr@?$basic_streambuf@DU?$char_
traits@D@std@@@std@@IEBAPEADXZ) referenced in function "protected: void __cdecl std::basic_stringbuf<char,struct std::c
har_traits<char>,class std::allocator<char> >::_Tidy(void)" (?_Tidy@?$basic_stringbuf@DU?$char_traits@D@std@@V?$allocat
or@D@2@@std@@IEAAXXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) protected: void __cdecl st
d::basic_streambuf<char,struct std::char_traits<char> >::setp(char *,char *)" (__imp_?setp@?$basic_streambuf@DU?$char_t
raits@D@std@@@std@@IEAAXPEAD0@Z) referenced in function "protected: void __cdecl std::basic_stringbuf<char,struct std::
char_traits<char>,class std::allocator<char> >::_Tidy(void)" (?_Tidy@?$basic_stringbuf@DU?$char_traits@D@std@@V?$alloca
tor@D@2@@std@@IEAAXXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) protected: void __cdecl st
d::basic_streambuf<char,struct std::char_traits<char> >::setp(char *,char *,char *)" (__imp_?setp@?$basic_streambuf@DU?
$char_traits@D@std@@@std@@IEAAXPEAD00@Z) referenced in function "protected: void __cdecl std::basic_stringbuf<char,stru
ct std::char_traits<char>,class std::allocator<char> >::_Init(char const *,unsigned __int64,int)" (?_Init@?$basic_strin
gbuf@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@IEAAXPEBD_KH@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) protected: char * __cdecl
std::basic_streambuf<char,struct std::char_traits<char> >::_Pninc(void)" (__imp_?_Pninc@?$basic_streambuf@DU?$char_trai
ts@D@std@@@std@@IEAAPEADXZ) referenced in function "protected: virtual int __cdecl std::basic_stringbuf<char,struct std
::char_traits<char>,class std::allocator<char> >::overflow(int)" (?overflow@?$basic_stringbuf@DU?$char_traits@D@std@@V?
$allocator@D@2@@std@@MEAAHH@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl st
d::basic_ios<char,struct std::char_traits<char> >::~basic_ios<char,struct std::char_traits<char> >(void)" (__imp_??1?$b
asic_ios@DU?$char_traits@D@std@@@std@@UEAA@XZ) referenced in function "int `public: __cdecl std::basic_istringstream<ch
ar,struct std::char_traits<char>,class std::allocator<char> >::basic_istringstream<char,struct std::char_traits<char>,c
lass std::allocator<char> >(class basic_istringstream<char,struct std::char_traits<char>,class std::allocator<char> >::
basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,int)'::`1'::dtor$0" (?dtor$0@?0???
0?$basic_istringstream@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$
allocator@D@2@@1@H@Z@4HA) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl std::
basic_ios<char,struct std::char_traits<char> >::setstate(int,bool)" (__imp_?setstate@?$basic_ios@DU?$char_traits@D@std@
@@std@@QEAAXH_N@Z) referenced in function "class std::basic_istream<char,struct std::char_traits<char> > & __cdecl std:
:getline<char,struct std::char_traits<char>,class std::allocator<char> >(class std::basic_istream<char,struct std::char
_traits<char> > &&,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > &,char)" (??
$getline@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@YAAEAV?$basic_istream@DU?$char_traits@D@std@@@0@$$QEAV10@AEAV?$b
asic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@0@D@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: class std::basic_s
treambuf<char,struct std::char_traits<char> > * __cdecl std::basic_ios<char,struct std::char_traits<char> >::rdbuf(void
)const " (__imp_?rdbuf@?$basic_ios@DU?$char_traits@D@std@@@std@@QEBAPEAV?$basic_streambuf@DU?$char_traits@D@std@@@2@XZ)
 referenced in function "class std::basic_istream<char,struct std::char_traits<char> > & __cdecl std::getline<char,stru
ct std::char_traits<char>,class std::allocator<char> >(class std::basic_istream<char,struct std::char_traits<char> > &&
,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > &,char)" (??$getline@DU?$char_
traits@D@std@@V?$allocator@D@2@@std@@YAAEAV?$basic_istream@DU?$char_traits@D@std@@@0@$$QEAV10@AEAV?$basic_string@DU?$ch
ar_traits@D@std@@V?$allocator@D@2@@0@D@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) protected: __cdecl std::ba
sic_ios<char,struct std::char_traits<char> >::basic_ios<char,struct std::char_traits<char> >(void)" (__imp_??0?$basic_i
os@DU?$char_traits@D@std@@@std@@IEAA@XZ) referenced in function "public: __cdecl std::basic_istringstream<char,struct s
td::char_traits<char>,class std::allocator<char> >::basic_istringstream<char,struct std::char_traits<char>,class std::a
llocator<char> >(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,int)"
(??0?$basic_istringstream@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@
V?$allocator@D@2@@1@H@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl std::basic
_istream<char,struct std::char_traits<char> >::basic_istream<char,struct std::char_traits<char> >(class std::basic_stre
ambuf<char,struct std::char_traits<char> > *,bool)" (__imp_??0?$basic_istream@DU?$char_traits@D@std@@@std@@QEAA@PEAV?$b
asic_streambuf@DU?$char_traits@D@std@@@1@_N@Z) referenced in function "public: __cdecl std::basic_istringstream<char,st
ruct std::char_traits<char>,class std::allocator<char> >::basic_istringstream<char,struct std::char_traits<char>,class
std::allocator<char> >(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,
int)" (??0?$basic_istringstream@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@QEAA@AEBV?$basic_string@DU?$char_traits@D
@std@@V?$allocator@D@2@@1@H@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl st
d::basic_istream<char,struct std::char_traits<char> >::~basic_istream<char,struct std::char_traits<char> >(void)" (__im
p_??1?$basic_istream@DU?$char_traits@D@std@@@std@@UEAA@XZ) referenced in function "int `public: __cdecl std::basic_istr
ingstream<char,struct std::char_traits<char>,class std::allocator<char> >::basic_istringstream<char,struct std::char_tr
aits<char>,class std::allocator<char> >(class basic_istringstream<char,struct std::char_traits<char>,class std::allocat
or<char> >::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,int)'::`1'::dtor$1" (?
dtor$1@?0???0?$basic_istringstream@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@QEAA@AEBV?$basic_string@DU?$char_trait
s@D@std@@V?$allocator@D@2@@1@H@Z@4HA) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: bool __cdecl std::
basic_istream<char,struct std::char_traits<char> >::_Ipfx(bool)" (__imp_?_Ipfx@?$basic_istream@DU?$char_traits@D@std@@@
std@@QEAA_N_N@Z) referenced in function "public: __cdecl std::basic_istream<char,struct std::char_traits<char> >::sentr
y::sentry(class std::basic_istream<char,struct std::char_traits<char> > &,bool)" (??0sentry@?$basic_istream@DU?$char_tr
aits@D@std@@@std@@QEAA@AEAV12@_N@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol __imp___std_init_once_begin_initialize referenced
 in function "void __cdecl std::call_once<class <lambda_0e64988457575224279dcb6d61e61bdd> >(struct std::once_flag &,cla
ss <lambda_0e64988457575224279dcb6d61e61bdd> &&)" (??$call_once@V<lambda_0e64988457575224279dcb6d61e61bdd>@@$$V@std@@YA
XAEAUonce_flag@0@$$QEAV<lambda_0e64988457575224279dcb6d61e61bdd>@@@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol __imp___std_init_once_complete referenced in func
tion "public: __cdecl std::_Init_once_completer::~_Init_once_completer(void)" (??1_Init_once_completer@std@@QEAA@XZ) [C
:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "int __cdecl Concurrency::details::_Schedule_chor
e(struct Concurrency::details::_Threadpool_chore *)" (?_Schedule_chore@details@Concurrency@@YAHPEAU_Threadpool_chore@12
@@Z) referenced in function "public: void __cdecl Concurrency::details::_DefaultPPLTaskScheduler::_PPLTaskChore::_Sched
ule(void)" (?_Schedule@_PPLTaskChore@_DefaultPPLTaskScheduler@details@Concurrency@@QEAAXXZ) [C:\Users\Sahil\SDL\build-w
inrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "void __cdecl Concurrency::details::_Release_chor
e(struct Concurrency::details::_Threadpool_chore *)" (?_Release_chore@details@Concurrency@@YAXPEAU_Threadpool_chore@12@
@Z) referenced in function "public: __cdecl Concurrency::details::_DefaultPPLTaskScheduler::_PPLTaskChore::~_PPLTaskCho
re(void)" (??1_PPLTaskChore@_DefaultPPLTaskScheduler@details@Concurrency@@QEAA@XZ) [C:\Users\Sahil\SDL\build-winrt-2nd\
SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "void __cdecl Concurrency::details::_ReportUnobse
rvedException(void)" (?_ReportUnobservedException@details@Concurrency@@YAXXZ) referenced in function "public: __cdecl C
oncurrency::details::_ExceptionHolder::~_ExceptionHolder(void)" (??1_ExceptionHolder@details@Concurrency@@QEAA@XZ) [C:\
Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "unsigned __int64 __cdecl Concurrency::details::p
latform::CaptureCallstack(void * *,unsigned __int64,unsigned __int64)" (?CaptureCallstack@platform@details@Concurrency@
@YA_KPEAPEAX_K1@Z) referenced in function "public: static class Concurrency::details::_TaskCreationCallstack __cdecl Co
ncurrency::details::_TaskCreationCallstack::_CaptureMultiFramesCallstack(void *,unsigned __int64)" (?_CaptureMultiFrame
sCallstack@_TaskCreationCallstack@details@Concurrency@@SA?AV123@PEAX_K@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxp
roj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "long __cdecl Concurrency::details::platform::Get
CurrentThreadId(void)" (?GetCurrentThreadId@platform@details@Concurrency@@YAJXZ) referenced in function "public: void _
_cdecl Concurrency::details::_CancellationTokenState::_DeregisterCallback(class Concurrency::details::_CancellationToke
nRegistration *)" (?_DeregisterCallback@_CancellationTokenState@details@Concurrency@@QEAAXPEAV_CancellationTokenRegistr
ation@23@@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "public: void __cdecl Concurrency::details::_Cont
extCallback::_CallInContext(class std::function<void __cdecl(void)>,bool)const " (?_CallInContext@_ContextCallback@deta
ils@Concurrency@@QEBAXV?$function@$$A6AXXZ@std@@_N@Z) referenced in function "public: void __cdecl Concurrency::details
::_ContextCallback::_CallInContext(class std::function<void __cdecl(void)>)const " (?_CallInContext@_ContextCallback@de
tails@Concurrency@@QEBAXV?$function@$$A6AXXZ@std@@@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "private: void __cdecl Concurrency::details::_Con
textCallback::_Reset(void)" (?_Reset@_ContextCallback@details@Concurrency@@AEAAXXZ) referenced in function "public: __c
decl Concurrency::details::_ContextCallback::~_ContextCallback(void)" (??1_ContextCallback@details@Concurrency@@QEAA@XZ
) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "private: void __cdecl Concurrency::details::_Con
textCallback::_Capture(void)" (?_Capture@_ContextCallback@details@Concurrency@@AEAAXXZ) referenced in function "public:
 static class Concurrency::details::_ContextCallback __cdecl Concurrency::details::_ContextCallback::_CaptureCurrent(vo
id)" (?_CaptureCurrent@_ContextCallback@details@Concurrency@@SA?AV123@XZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxp
roj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "private: void __cdecl Concurrency::details::_Exc
eptionHolder::ReportUnhandledError(void)" (?ReportUnhandledError@_ExceptionHolder@details@Concurrency@@AEAAXXZ) referen
ced in function "public: __cdecl Concurrency::details::_ExceptionHolder::~_ExceptionHolder(void)" (??1_ExceptionHolder@
details@Concurrency@@QEAA@XZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "public: void __cdecl Concurrency::details::_Task
EventLogger::_LogScheduleTask(bool)" (?_LogScheduleTask@_TaskEventLogger@details@Concurrency@@QEAAX_N@Z) referenced in
function "private: void __cdecl Concurrency::task<unsigned char>::_TaskInitWithFunctor<void,class <lambda_ec480422d35a7
9845c9721bfeb0eab68> >(class <lambda_ec480422d35a79845c9721bfeb0eab68> const &)" (??$_TaskInitWithFunctor@XV<lambda_ec4
80422d35a79845c9721bfeb0eab68>@@@?$task@E@Concurrency@@AEAAXAEBV<lambda_ec480422d35a79845c9721bfeb0eab68>@@@Z) [C:\User
s\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "public: void __cdecl Concurrency::details::_Task
EventLogger::_LogCancelTask(void)" (?_LogCancelTask@_TaskEventLogger@details@Concurrency@@QEAAXXZ) referenced in functi
on "public: virtual bool __cdecl Concurrency::details::_Task_impl<unsigned char>::_CancelAndRunContinuations(bool,bool,
bool,class std::shared_ptr<struct Concurrency::details::_ExceptionHolder> const &)" (?_CancelAndRunContinuations@?$_Tas
k_impl@E@details@Concurrency@@UEAA_N_N00AEBV?$shared_ptr@U_ExceptionHolder@details@Concurrency@@@std@@@Z) [C:\Users\Sah
il\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "public: void __cdecl Concurrency::details::_Task
EventLogger::_LogTaskCompleted(void)" (?_LogTaskCompleted@_TaskEventLogger@details@Concurrency@@QEAAXXZ) referenced in
function "public: virtual __cdecl Concurrency::details::_PPLTaskHandle<unsigned char,struct Concurrency::task<unsigned
char>::_InitialTaskHandle<void,class <lambda_ec480422d35a79845c9721bfeb0eab68>,struct Concurrency::details::_TypeSelect
orNoAsync>,struct Concurrency::details::_TaskProcHandle>::~_PPLTaskHandle<unsigned char,struct Concurrency::task<unsign
ed char>::_InitialTaskHandle<void,class <lambda_ec480422d35a79845c9721bfeb0eab68>,struct Concurrency::details::_TypeSel
ectorNoAsync>,struct Concurrency::details::_TaskProcHandle>(void)" (??1?$_PPLTaskHandle@EU?$_InitialTaskHandle@XV<lambd
a_ec480422d35a79845c9721bfeb0eab68>@@U_TypeSelectorNoAsync@details@Concurrency@@@?$task@E@Concurrency@@U_TaskProcHandle
@details@3@@details@Concurrency@@UEAA@XZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "public: void __cdecl Concurrency::details::_Task
EventLogger::_LogTaskExecutionCompleted(void)" (?_LogTaskExecutionCompleted@_TaskEventLogger@details@Concurrency@@QEAAX
XZ) referenced in function "public: virtual void __cdecl Concurrency::details::_PPLTaskHandle<unsigned char,struct Conc
urrency::task<unsigned char>::_InitialTaskHandle<void,class <lambda_ec480422d35a79845c9721bfeb0eab68>,struct Concurrenc
y::details::_TypeSelectorNoAsync>,struct Concurrency::details::_TaskProcHandle>::invoke(void)const " (?invoke@?$_PPLTas
kHandle@EU?$_InitialTaskHandle@XV<lambda_ec480422d35a79845c9721bfeb0eab68>@@U_TypeSelectorNoAsync@details@Concurrency@@
@?$task@E@Concurrency@@U_TaskProcHandle@details@3@@details@Concurrency@@UEBAXXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SD
L2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "public: void __cdecl Concurrency::details::_Task
EventLogger::_LogWorkItemStarted(void)" (?_LogWorkItemStarted@_TaskEventLogger@details@Concurrency@@QEAAXXZ) referenced
 in function "public: __cdecl Concurrency::details::_TaskWorkItemRAIILogger::_TaskWorkItemRAIILogger(struct Concurrency
::details::_TaskEventLogger &)" (??0_TaskWorkItemRAIILogger@details@Concurrency@@QEAA@AEAU_TaskEventLogger@12@@Z) [C:\U
sers\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol "public: void __cdecl Concurrency::details::_Task
EventLogger::_LogWorkItemCompleted(void)" (?_LogWorkItemCompleted@_TaskEventLogger@details@Concurrency@@QEAAXXZ) refere
nced in function "public: __cdecl Concurrency::details::_TaskWorkItemRAIILogger::~_TaskWorkItemRAIILogger(void)" (??1_T
askWorkItemRAIILogger@details@Concurrency@@QEAA@XZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2001: unresolved external symbol "public: virtual void __cdecl std::basic_streambu
f<char,struct std::char_traits<char> >::_Lock(void)" (?_Lock@?$basic_streambuf@DU?$char_traits@D@std@@@std@@UEAAXXZ) [C
:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2001: unresolved external symbol "public: virtual void __cdecl std::basic_streambu
f<char,struct std::char_traits<char> >::_Unlock(void)" (?_Unlock@?$basic_streambuf@DU?$char_traits@D@std@@@std@@UEAAXXZ
) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2001: unresolved external symbol "protected: virtual void __cdecl std::basic_strea
mbuf<char,struct std::char_traits<char> >::imbue(class std::locale const &)" (?imbue@?$basic_streambuf@DU?$char_traits@
D@std@@@std@@MEAAXAEBVlocale@2@@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2001: unresolved external symbol "protected: virtual class std::basic_streambuf<ch
ar,struct std::char_traits<char> > * __cdecl std::basic_streambuf<char,struct std::char_traits<char> >::setbuf(char *,_
_int64)" (?setbuf@?$basic_streambuf@DU?$char_traits@D@std@@@std@@MEAAPEAV12@PEAD_J@Z) [C:\Users\Sahil\SDL\build-winrt-2
nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2001: unresolved external symbol "protected: virtual __int64 __cdecl std::basic_st
reambuf<char,struct std::char_traits<char> >::showmanyc(void)" (?showmanyc@?$basic_streambuf@DU?$char_traits@D@std@@@st
d@@MEAA_JXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2001: unresolved external symbol "protected: virtual int __cdecl std::basic_stream
buf<char,struct std::char_traits<char> >::sync(void)" (?sync@?$basic_streambuf@DU?$char_traits@D@std@@@std@@MEAAHXZ) [C
:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2001: unresolved external symbol "protected: virtual int __cdecl std::basic_stream
buf<char,struct std::char_traits<char> >::uflow(void)" (?uflow@?$basic_streambuf@DU?$char_traits@D@std@@@std@@MEAAHXZ)
[C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2001: unresolved external symbol "protected: virtual __int64 __cdecl std::basic_st
reambuf<char,struct std::char_traits<char> >::xsgetn(char *,__int64)" (?xsgetn@?$basic_streambuf@DU?$char_traits@D@std@
@@std@@MEAA_JPEAD_J@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2001: unresolved external symbol "protected: virtual __int64 __cdecl std::basic_st
reambuf<char,struct std::char_traits<char> >::xsputn(char const *,__int64)" (?xsputn@?$basic_streambuf@DU?$char_traits@
D@std@@@std@@MEAA_JPEBD_J@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_sysfilesystem.obj : error LNK2001: unresolved external symbol _CxxThrowException [C:\Users\Sahil\SDL\build-winrt-2n
d\SDL2.vcxproj]
msvcrtd.lib(throw_bad_alloc.obj) : error LNK2001: unresolved external symbol _CxxThrowException [C:\Users\Sahil\SDL\bui
ld-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2001: unresolved external symbol _CxxThrowException [C:\Users\Sahil\SDL\build-winr
t-2nd\SDL2.vcxproj]
SDL_wasapi_winrt.obj : error LNK2001: unresolved external symbol _CxxThrowException [C:\Users\Sahil\SDL\build-winrt-2nd
\SDL2.vcxproj]
SDL_winrtmessagebox.obj : error LNK2001: unresolved external symbol _CxxThrowException [C:\Users\Sahil\SDL\build-winrt-
2nd\SDL2.vcxproj]
SDL_winrtvideo.obj : error LNK2001: unresolved external symbol _CxxThrowException [C:\Users\Sahil\SDL\build-winrt-2nd\S
DL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol memcmp referenced in function "public: static int
 __cdecl std::_Narrow_char_traits<char,int>::compare(char const * const,char const * const,unsigned __int64)" (?compare
@?$_Narrow_char_traits@DH@std@@SAHQEBD0_K@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_winrtapp_direct3d.obj : error LNK2019: unresolved external symbol memmove referenced in function "void * * __cdecl
std::_Copy_memmove<void * *,void * *>(void * *,void * *,void * *)" (??$_Copy_memmove@PEAPEAXPEAPEAX@std@@YAPEAPEAXPEAPE
AX00@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_sysfilesystem.obj : error LNK2001: unresolved external symbol memmove [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxp
roj]
SDL_sysurl.obj : error LNK2019: unresolved external symbol __Platform_wcslen referenced in function "public: __cdecl Pl
atform::String::String(wchar_t const *)" (??0String@Platform@@QE$AAA@PEB_W@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.
vcxproj]
SDL_winrtmessagebox.obj : error LNK2001: unresolved external symbol __Platform_wcslen [C:\Users\Sahil\SDL\build-winrt-2
nd\SDL2.vcxproj]
vccorlibd.lib(__Platform_wcslen.obj) : error LNK2001: unresolved external symbol __Platform_wcslen [C:\Users\Sahil\SDL\
build-winrt-2nd\SDL2.vcxproj]
SDL_syslocale.obj : error LNK2019: unresolved external symbol GetModuleHandle referenced in function SDL_SYS_GetPreferr
edLocales [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
SDL_sysfilesystem.obj : error LNK2019: unresolved external symbol ceilf referenced in function "protected: unsigned __i
nt64 __cdecl std::_Hash<class std::_Umap_traits<enum SDL_WinRT_Path,class std::basic_string<char,struct std::char_trait
s<char>,class std::allocator<char> >,class std::_Uhash_compare<enum SDL_WinRT_Path,struct std::hash<enum SDL_WinRT_Path
>,struct std::equal_to<enum SDL_WinRT_Path> >,class std::allocator<struct std::pair<enum SDL_WinRT_Path const ,class st
d::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > > >,0> >::_Min_load_factor_buckets(unsi
gned __int64)const " (?_Min_load_factor_buckets@?$_Hash@V?$_Umap_traits@W4SDL_WinRT_Path@@V?$basic_string@DU?$char_trai
ts@D@std@@V?$allocator@D@2@@std@@V?$_Uhash_compare@W4SDL_WinRT_Path@@U?$hash@W4SDL_WinRT_Path@@@std@@U?$equal_to@W4SDL_
WinRT_Path@@@3@@3@V?$allocator@U?$pair@$$CBW4SDL_WinRT_Path@@V?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@s
td@@@std@@@3@$0A@@std@@@std@@IEBA_K_K@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
vccorlibd.lib(init.obj) : error LNK2019: unresolved external symbol __imp_wcscmp referenced in function "public: void _
_cdecl <lambda_3239fb845f78de647d1691047489f1a6>::operator()(struct Microsoft::WRL::Details::CreatorMap const *,struct
Microsoft::WRL::Details::CreatorMap const *)const " (??R<lambda_3239fb845f78de647d1691047489f1a6>@@QEBAXPEBUCreatorMap@
Details@WRL@Microsoft@@0@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
vccorlibd.lib(init.obj) : error LNK2019: unresolved external symbol __imp_wcslen referenced in function "private: void
__cdecl Microsoft::WRL::Module<1,class Platform::Details::InProcModule>::VerifyEntries(void)" (?VerifyEntries@?$Module@
$00VInProcModule@Details@Platform@@@WRL@Microsoft@@AEAAXXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
vccorlibd.lib(__Platform_wcslen.obj) : error LNK2001: unresolved external symbol wcslen [C:\Users\Sahil\SDL\build-winrt
-2nd\SDL2.vcxproj]
msvcrtd.lib(ehvecdtr.obj) : error LNK2019: unresolved external symbol terminate referenced in function "int __cdecl Arr
ayUnwindFilter(struct _EXCEPTION_POINTERS *)" (?ArrayUnwindFilter@@YAHPEAU_EXCEPTION_POINTERS@@@Z) [C:\Users\Sahil\SDL\
build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(ehvecdtr.obj) : error LNK2019: unresolved external symbol __current_exception referenced in function "int _
_cdecl ArrayUnwindFilter(struct _EXCEPTION_POINTERS *)" (?ArrayUnwindFilter@@YAHPEAU_EXCEPTION_POINTERS@@@Z) [C:\Users\
Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(ehvecdtr.obj) : error LNK2019: unresolved external symbol __current_exception_context referenced in functio
n "int __cdecl ArrayUnwindFilter(struct _EXCEPTION_POINTERS *)" (?ArrayUnwindFilter@@YAHPEAU_EXCEPTION_POINTERS@@@Z) [C
:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(ehvecdtr.obj) : error LNK2001: unresolved external symbol __C_specific_handler [C:\Users\Sahil\SDL\build-wi
nrt-2nd\SDL2.vcxproj]
msvcrtd.lib(utility.obj) : error LNK2001: unresolved external symbol __C_specific_handler [C:\Users\Sahil\SDL\build-win
rt-2nd\SDL2.vcxproj]
msvcrtd.lib(dll_dllmain.obj) : error LNK2001: unresolved external symbol __C_specific_handler [C:\Users\Sahil\SDL\build
-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(new_scalar.obj) : error LNK2019: unresolved external symbol _callnewh referenced in function "void * __cdec
l operator new(unsigned __int64)" (??2@YAPEAX_K@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(new_scalar.obj) : error LNK2019: unresolved external symbol malloc referenced in function "void * __cdecl o
perator new(unsigned __int64)" (??2@YAPEAX_K@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(utility.obj) : error LNK2019: unresolved external symbol _seh_filter_dll referenced in function __scrt_dllm
ain_exception_filter [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(utility.obj) : error LNK2019: unresolved external symbol _configure_narrow_argv referenced in function "pub
lic: static int __cdecl __scrt_narrow_argv_policy::configure_argv(void)" (?configure_argv@__scrt_narrow_argv_policy@@SA
HXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(utility.obj) : error LNK2019: unresolved external symbol _initialize_narrow_environment referenced in funct
ion "public: static int __cdecl __scrt_narrow_environment_policy::initialize_environment(void)" (?initialize_environmen
t@__scrt_narrow_environment_policy@@SAHXZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(utility.obj) : error LNK2019: unresolved external symbol _initialize_onexit_table referenced in function __
scrt_initialize_onexit_tables [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(utility.obj) : error LNK2019: unresolved external symbol _register_onexit_function referenced in function _
onexit [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(utility.obj) : error LNK2019: unresolved external symbol _execute_onexit_table referenced in function __scr
t_dllmain_uninitialize_c [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(utility.obj) : error LNK2019: unresolved external symbol _crt_atexit referenced in function _onexit [C:\Use
rs\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(utility.obj) : error LNK2019: unresolved external symbol _crt_at_quick_exit referenced in function at_quick
_exit [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(utility.obj) : error LNK2019: unresolved external symbol _cexit referenced in function __scrt_dllmain_unini
tialize_c [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(dll_dllmain.obj) : error LNK2019: unresolved external symbol _initterm referenced in function "int __cdecl
dllmain_crt_process_attach(struct HINSTANCE__ * const,void * const)" (?dllmain_crt_process_attach@@YAHQEAUHINSTANCE__@@
QEAX@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(dll_dllmain.obj) : error LNK2019: unresolved external symbol _initterm_e referenced in function "int __cdec
l dllmain_crt_process_attach(struct HINSTANCE__ * const,void * const)" (?dllmain_crt_process_attach@@YAHQEAUHINSTANCE__
@@QEAX@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(init.obj) : error LNK2019: unresolved external symbol _CrtDbgReport referenced in function _CRT_RTC_INIT [C
:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(init.obj) : error LNK2019: unresolved external symbol _CrtDbgReportW referenced in function _CRT_RTC_INITW
[C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(delete_scalar.obj) : error LNK2019: unresolved external symbol _free_dbg referenced in function "void __cde
cl operator delete(void *)" (??3@YAXPEAX@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(tncleanup.obj) : error LNK2019: unresolved external symbol __std_type_info_destroy_list referenced in funct
ion "void __cdecl __scrt_uninitialize_type_info(void)" (?__scrt_uninitialize_type_info@@YAXXZ) [C:\Users\Sahil\SDL\buil
d-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(error.obj) : error LNK2019: unresolved external symbol strcpy_s referenced in function "void __cdecl _RTC_S
tackFailure(void *,char const *)" (?_RTC_StackFailure@@YAXPEAXPEBD@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(error.obj) : error LNK2019: unresolved external symbol strcat_s referenced in function "void __cdecl _RTC_S
tackFailure(void *,char const *)" (?_RTC_StackFailure@@YAXPEAXPEBD@Z) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(error.obj) : error LNK2019: unresolved external symbol __stdio_common_vsprintf_s referenced in function _vs
printf_s_l [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(error.obj) : error LNK2001: unresolved external symbol __C_specific_handler_noexcept [C:\Users\Sahil\SDL\bu
ild-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(pdblkup.obj) : error LNK2019: unresolved external symbol _wmakepath_s referenced in function "int __cdecl G
etPdbDllPathFromFilePath(wchar_t const *,wchar_t *,unsigned __int64)" (?GetPdbDllPathFromFilePath@@YAHPEB_WPEA_W_K@Z) [
C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(pdblkup.obj) : error LNK2019: unresolved external symbol _wsplitpath_s referenced in function "int __cdecl
GetPdbDllPathFromFilePath(wchar_t const *,wchar_t *,unsigned __int64)" (?GetPdbDllPathFromFilePath@@YAHPEB_WPEA_W_K@Z)
[C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(pdblkup.obj) : error LNK2019: unresolved external symbol wcscpy_s referenced in function "int __cdecl GetPd
bDllPathFromFilePath(wchar_t const *,wchar_t *,unsigned __int64)" (?GetPdbDllPathFromFilePath@@YAHPEB_WPEA_W_K@Z) [C:\U
sers\Sahil\SDL\build-winrt-2nd\SDL2.vcxproj]
msvcrtd.lib(pdblkup.obj) : error LNK2019: unresolved external symbol __vcrt_GetModuleFileNameW referenced in function "
struct HINSTANCE__ * __cdecl GetPdbDll(void)" (?GetPdbDll@@YAPEAUHINSTANCE__@@XZ) [C:\Users\Sahil\SDL\build-winrt-2nd\S
DL2.vcxproj]
msvcrtd.lib(pdblkup.obj) : error LNK2019: unresolved external symbol __vcrt_GetModuleHandleW referenced in function "st
ruct HINSTANCE__ * __cdecl GetPdbDll(void)" (?GetPdbDll@@YAPEAUHINSTANCE__@@XZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL
2.vcxproj]
msvcrtd.lib(pdblkup.obj) : error LNK2019: unresolved external symbol __vcrt_LoadLibraryExW referenced in function "stru
ct HINSTANCE__ * __cdecl GetPdbDll(void)" (?GetPdbDll@@YAPEAUHINSTANCE__@@XZ) [C:\Users\Sahil\SDL\build-winrt-2nd\SDL2.
vcxproj]
C:\Users\Sahil\SDL\build-winrt-2nd\Debug\SDL2d.dll : fatal error LNK1120: 117 unresolved externals [C:\Users\Sahil\SDL\
build-winrt-2nd\SDL2.vcxproj]
```

Am I missing anything or any steps?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,15 +6,8 @@ cmake_minimum_required(VERSION 3.0.0)
 project(SDL2 C CXX)
 
 if(WINDOWS_STORE)
-  enable_language(CXX)
   cmake_minimum_required(VERSION 3.11)
   add_definitions(-DSDL_BUILDING_WINRT=1 -ZW)
-  link_libraries(
-    -nodefaultlib:vccorlib$<$<CONFIG:Debug>:d>
-    -nodefaultlib:msvcrt$<$<CONFIG:Debug>:d>
-    vccorlib$<$<CONFIG:Debug>:d>.lib
-    msvcrt$<$<CONFIG:Debug>:d>.lib
-  )
 endif()
 
 # !!! FIXME: this should probably do "MACOSX_RPATH ON" as a target property
@@ -1576,7 +1569,11 @@ elseif(WINDOWS)
   endif()
 
   if(SDL_LOCALE)
-    file(GLOB LOCALE_SOURCES ${SDL2_SOURCE_DIR}/src/locale/windows/*.c)
+    if(WINDOWS_STORE)
+      file(GLOB LOCALE_SOURCES ${SDL2_SOURCE_DIR}/src/locale/winrt/*.c)
+    else()
+      file(GLOB LOCALE_SOURCES ${SDL2_SOURCE_DIR}/src/locale/windows/*.c)
+    endif()
     set(SOURCE_FILES ${SOURCE_FILES} ${LOCALE_SOURCES})
     set(HAVE_SDL_LOCALE TRUE)
   endif()
@@ -1595,6 +1592,15 @@ elseif(WINDOWS)
   # Libraries for Win32 native and MinGW
   if(NOT WINDOWS_STORE)
     list(APPEND EXTRA_LIBS user32 gdi32 winmm imm32 ole32 oleaut32 version uuid advapi32 setupapi shell32)
+  endif()
+
+  if(WINDOWS_STORE)
+    list(APPEND EXTRA_LIBS
+      -nodefaultlib:vccorlib$<$<CONFIG:Debug>:d>
+      -nodefaultlib:msvcrt$<$<CONFIG:Debug>:d>
+      vccorlib$<$<CONFIG:Debug>:d>.lib
+      msvcrt$<$<CONFIG:Debug>:d>.lib
+    )
   endif()
 
   # TODO: in configure.ac the check for timers is set on
@@ -2406,8 +2412,10 @@ if(SDL_SHARED)
   endif()
   if(MSVC AND NOT LIBC)
     # Don't try to link with the default set of libraries.
-    set_target_properties(SDL2 PROPERTIES LINK_FLAGS_RELEASE "/NODEFAULTLIB")
-    set_target_properties(SDL2 PROPERTIES LINK_FLAGS_DEBUG "/NODEFAULTLIB")
+    if(NOT WINDOWS_STORE)
+      set_target_properties(SDL2 PROPERTIES LINK_FLAGS_RELEASE "/NODEFAULTLIB")
+      set_target_properties(SDL2 PROPERTIES LINK_FLAGS_DEBUG "/NODEFAULTLIB")
+    endif()
     set_target_properties(SDL2 PROPERTIES STATIC_LIBRARY_FLAGS "/NODEFAULTLIB")
   endif()
   set(_INSTALL_LIBS "SDL2" ${_INSTALL_LIBS})

